support/remote_health_monitoring/insights-operator-simple-access.adoc

```diff
@@ -0,0 +1,20 @@
+[id="insights-operator-simple-access"]
+= Importing RHEL Simple Content Access certificates with Insights Operator
+include::modules/common-attributes.adoc[]
+:context: remote-health-reporting-from-restricted-network
+:FeatureName: `InsightsOperatorPullingSCA`
+
+toc::[]
+
+Insights Operator can import your RHEL Simple Content Access (SCA) certificates from {cloud-redhat-com}. Simple Content Access is a capability in Red Hat’s subscription tools which simplifies the behavior of the entitlement tooling, making it easier to consume the content provided by your Red Hat subscriptions without the complexity of configuring subscription tooling. After importing the certificates, they are stored in the `etc-pki-entitlement` secret in the `openshift-config-managed` namespace. 
```

Comment:
```suggestion
Insights Operator can import your RHEL Simple Content Access (SCA) certificates from {cloud-redhat-com}. SCA is a capability in Red Hat’s subscription tools which simplifies the behavior of the entitlement tooling. It is easier to consume the content provided by your Red Hat subscriptions without the complexity of configuring subscription tooling. After importing the certificates, they are stored in the `etc-pki-entitlement` secret in the `openshift-config-managed` namespace. 
```
Simplified the second sentence a little bit for translation. If it changes the meaning of the sentence, please ignore.

Reply:
> Just a few nit things. Looks good overall.
> 
> House keeping question: Is this content _only_ for 4.9, meaning this will not be supported in 4.10? Or should this go into the main branch starting with 4.9? You are currently only including it in `enterprise-4.9` branch meaning it won't be included in 4.10.

You are right - I've confirmed with the developer team that this feature will be in future releases, so we should move this PR (or recreate it) to be merged into the `main` branch, and cherry-pick to 4.9.